sdk/schemaregistry/schema-registry-avro/src/logicalTypes.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
should we name this module datetime instead and place it under a logicalTypes folder?